scripts/test_critical_features.py

```
@@ -73,6 +73,12 @@ def config():
     return config
 
 
+@pytest.fixture
+def state():
+    """Create a test state."""
+    return AppState()
```

Ruff (0.15.1)

[error] Undefined name `AppState`

(F821)